.github/workflows/linters.yml

```diff
@@ -0,0 +1,23 @@
+name: Fast DDS Linters CI
+
+on:
+  pull_request:
+    types:
+      - review_requested
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - '!**/CMakeLists.txt'
```

Comment:
If we are going to have the linters workflow as required for merging, it is better to always run it.

```suggestion
```

Reply:
Alright. I have seen that is other repos already using the action is not a problem.